examples/v2/gke/crd/api-options.yaml

```diff
@@ -0,0 +1,16 @@
+options:
+  validationOptions:
+    schemaValidation: IGNORE_WITH_WARNINGS
```

Comment:
I tried this with FAIL as well, and was able to successfully push my demo. YMMV.

Reply:
added `Fail` to both and worked fine still